setup.py

```diff
@@ -21,6 +21,7 @@
     "numpy",
     "pandas",
     "xarray",
+    "pangeo-xesmf",
```

Comment:
We should delete the `install_requires` section in `setup.py` so that we don't use PyPI dependencies in our conda-env environments.

I foudn that the xEMSF release on PyPI (`pangeo-xesmf`) includes `numba` as a sub-dependency, while the conda-forge version (`xemsf`) does not. This means the conda-forge version breaks if `numba` isn't included in the environment or as a dependency of `xcdat`.

https://github.com/pangeo-data/xESMF/issues/170